code/_onclick/item_attack.dm

```diff
@@ -70,7 +70,8 @@
 
 	user.do_attack_animation(M)
 	M.attacked_by(src, user)
-
+	if(force)
+		M.damage_record += src

```

Comment:
what about (handless) simplemobs and punches? shouldn't they also get logged?

Reply:
oh yeah right i have forgotten bout those

Reply:
implemented